src/apiCalls.js

```diff
@@ -6,9 +9,23 @@ let pageData = {
   currentRecipeCard: {}
 };
 
-// Your fetch requests will live here!
+// API CALLS
+
+const assignCurrentUser = () => {
+  fetch('https://what-s-cookin-starter-kit.herokuapp.com/api/v1/users')
+    .then(response => response.json())
+    .then(users => {
+      currentUser = getRandomUser(users.users);
+    })
+    .catch(error => {
+      console.error(error)
+    })
+}
+
+const updateCurrentUser = (user) => {
```

Comment:
given that we have assignCurrentUser, why would we need updateCurrentUser after a pageLoad. My question stems from an understanding that we only assign a user on pageLoad and assignCurrentUser does that. 
Am I missing something?

Reply:
Our addRecipesToCook function is a pure function that returns the new user and we need to reassign currentUser to the user returned by the pure function. Given the error `cannot update properties of #<Object> which has only a getter` in order to use the pure function, we will need to use this function to reassign current User. In a following iteration, when adding recipes to cook, we will say `updateCurrentUser(addRecipesToCook(currentUser, recipe))`

Reply:
RIGHT! You DID explain this. Thank you. Sorry for not recalling it fully. Appreciate you looking out for this nitty gritties.